eng/pipelines/templates/steps/use-node-test-version.yml

```diff
@@ -24,6 +24,12 @@ steps:
         # Map from the symlink path to the target path (npm has issues installing into symlink dirs)
         # Example: common/temp/node_modules/.pnpm/keytar@5.6.0/node_modules/keytar
         $targetPath = (Get-Item $symlink).Target
+        Write-Host "Target of symlink : $($targetPath)"
+        if ($targetPath.StartsWith(".."))
```

Comment:
Is there a reason to not just always resolve the path?

Reply:
We are resolving the path few lines below when this path is parsed. I have done in this added step to get a more readable path in log.

Reply:
There should be a better way to get the absolute path of the symlink target.  I'm trying to figure this out now.

Reply:
The following code should work on both Powershell 7.1 and 7.2:

```powershell
$symlinkInfo = Get-Item $symlink
$targetPath = [IO.Path]::Combine($symlinkInfo.Parent, $symlinkInfo.Target)
```